ush/eva/jedi_gsi_compare_conv.yaml

```diff
@@ -368,18 +371,19 @@ diagnostics:
         - add_xlabel: 'h(x) JEDI - h(x) GSI'
           add_ylabel: 'Count'
           statistics:
-            data:
-              variable: experiment::HofXDiff::${variable}
-              @CHANNELKEY@
+            fields:
+              - field_name: experiment::HofXDiff::${variable}
+                @CHANNELKEY@
```

Comment:
Looking over the previous commit that @CoryMartin-NOAA, it looks like this should be 
`                channel: ${channel}`
but I might be wrong. What is the proper way to do this?

Reply:
This is because the python script decides to put that in if it needs to